plugins/hls-tactics-plugin/test/Utils.hs

```diff
@@ -8,26 +8,30 @@
 module Utils where
 
 import           Control.Applicative.Combinators (skipManyTill)
-import           Control.Lens hiding (failing, (<.>))
-import           Control.Monad (unless)
+import           Control.Lens                    hiding (failing, (.=), (<.>))
```

Comment:
Please don't adjust the formatting of this file

Reply:
This was done by the pre-commit hook. I will revert it.

Reply:
Thanks. @Ailrun, is there a way we can disable the pre-commit hook for Wingman files?